src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/MultipleUtil.java

```diff
@@ -67,6 +67,9 @@
 import static uk.gov.hmcts.et.common.model.multiples.MultipleConstants.HEADER_4;
 import static uk.gov.hmcts.et.common.model.multiples.MultipleConstants.SHEET_NAME;
 
+@SuppressWarnings({"PMD.LooseCoupling", "PMD.LawOfDemeter", "PMD.LinguisticNaming",
+    "PMD.ExcessiveMethodLength", "PMD.UseUnderscoresInNumericLiterals", "PMD.JUnit5TestShouldBePackagePrivate",
+    "PMD.ConfusingTernary", "PMD.CloseResource"})
```

Comment:
PMD.JUnit5TestShouldBePackagePrivate

Why is this even in here??? This isn't a test class!!!

Reply:
Yes it is not a test class. But it already exist in the test project and there are test classes that depend on it. Adjusting the suppressions and some of them are required.